src/main/java/ro/siit/Product.java

```diff
@@ -7,10 +7,16 @@
 
     private String name;
     private BigDecimal price;
+    private double discount;
```

Comment:
Maybe it would have been better to use `BigDecimal` here.

Reply:
exagerati sefu'